test/test_facebook.py

```diff
@@ -151,14 +154,22 @@ def test_invalid_format(self):
 
 class TestAuthURL(FacebookTestCase):
     def test_auth_url(self):
+        default_version = facebook.VALID_API_VERSIONS[0]
+        api = facebook.GraphAPI(version=default_version)
```

Comment:
It's not necessary to pull a version from `VALID_API_VERSIONS` here - if you initialize a `GraphAPI` object without specifying a version, we already use `VALID_API_VERSIONS[0]` - see https://github.com/mobolic/facebook-sdk/blob/08bb2773cd69fe8b996d0dc10f811fbbd21ca945/facebook/__init__.py#L104.